scala-cli.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -eu
+
+# This script automatically download scala-cli from latest release and cache via coursier
+
+# https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c
+LATATEST_TAG=$(
+    curl --silent "https://api.github.com/repos/VirtusLab/scala-cli/releases/latest" | grep '"tag_name":' | sed 's/.*"\(v.*\)",/\1/'
```

Comment:
It seems that means every run of this script will run `curl` command upfront… I don't think we should do that. (If `cs` was installed, we could use `cs get --changing` instead of curl, and benefit from caching. But let's not assume `cs` is installed either.)

We should rather hardcode the latest Scala CLI version here, and automatically update it from the CI upon new releases.